tests/Service/FunctionsTest.php

```diff
@@ -0,0 +1,113 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Service;
+
+use Lmc\ApiFilter\AbstractTestCase;
+use Lmc\ApiFilter\Entity\Value;
+use Lmc\ApiFilter\Exception\InvalidArgumentException;
+use Lmc\ApiFilter\Filter\FunctionParameter;
+
+/**
+ * @group unit
+ */
+class FunctionsTest extends AbstractTestCase
+{
+    /** @var Functions */
+    private $functions;
+
+    protected function setUp(): void
+    {
+        $this->functions = new Functions();
+    }
+
+    /**
+     * @test
+     */
+    public function shouldRegisterFullNameFunction(): void
+    {
+        $sql = 'SELECT * FROM person WHERE firstName = \':firstName\' AND surname = \':surname\'';
```

Comment:
Why are you escaping the string? You can just use `"SELECT * ...  firstName = ':firstName'";`